bin/package-browser

```diff
@@ -56,19 +56,8 @@ if [ $CI != true ]; then
     yarn lint
 fi
 
-(
-  cd packages/loot-design;
-  ../../node_modules/.bin/patch-package
-)
```

Comment:
Looks like `patch-package` isn't running anymore

Reply:
The downshift patch was the only one which was being run in the `loot-design` context but that's been moved up to the monorepo root so this is unneeded now.

Reply:
I moved the patch file up as the dependency was now stored in the monorepo root's version of `node_modules` and so the patch moved to match.